pkg/flag/kubernetes_flags.go

```diff
@@ -79,6 +79,13 @@ var (
 		Default:    "trivy-temp",
 		Usage:      "specify the namespace in which the node-collector job should be deployed",
 	}
+	ExcludeOwned = Flag{
+		Name:       "exclude-owned",
+		ConfigName: "kubernetes.exclude.owned",
+		Shorthand:  "E",
+		Default:    false,
+		Usage:      "exclude objects that have an owner reference",
```

Comment:
maybe replace the wording `object` with `resource`

Reply:
Done.